pyo3-ffi/src/cpython/critical_section.rs

```diff
@@ -9,13 +9,47 @@ pub struct PyCriticalSection {
     _cs_mutex: *mut PyMutex,
 }
 
+#[cfg(Py_GIL_DISABLED)]
+impl PyCriticalSection {
+    pub const fn new() -> PyCriticalSection {
+        PyCriticalSection {
+            _cs_prev: 0,
+            _cs_mutex: std::ptr::null_mut(),
+        }
+    }
+}
+
+#[cfg(Py_GIL_DISABLED)]
+impl Default for PyCriticalSection {
+    fn default() -> Self {
+        PyCriticalSection::new()
+    }
+}
```

Comment:
I guess all these new / default values weren't needed, as you used `mem::zeroed`?

Reply:
I pushed a commit to revert these for now.